load_testing/common_flows/flow_helper.py

```diff
@@ -9,6 +9,19 @@
 def do_request(
     context, method, path, expected_redirect=None, data={}, files={}, name=None
 ):
+
```

Comment:
Can we move this into the error handler only?   This will blast one line per-request which will murder us.

Reply:
Well I just moved the method up here :). There's probably a better way, so please feel free to improve this :)